### ISSUE TITLE
Possible bug for [TermAttribute] with multiple params values 

### ISSUE THREAD

**Issue by mamift** [atata-framework/atata#799]
Hello

So I'm trying to model possible `<option>` values for a `<select>` element. The HTML is basically:

```HTML
<select data-unique-identifier="3fae10ba-dbe5-432f-a4b5-e031f699fa3c" class="status">
  <option value="TODO" selected="">To do</option>
  <!-- Can also be: <option value="To do">To do</option> -->
  <option value="COMPLETED">Completed</option>
  <!-- Can also be: <option value="Completed">Completed</option> -->
  <option value="WAIVED">Waived</option>
  <!-- Can also be: <option value="Waived">Waived</option> -->
</select>
```

The enum I've written is:
```C#
public enum BringUpStatusType
{
  [Term(TermMatch.Equals, "TODO", "To do")]
  ToDo,
  [Term(TermMatch.Equals, "COMPLETED", nameof(Completed))]
  Completed,
  [Term(TermMatch.Equals, "WAIVED", nameof(Waived))]
  Waived
}

// is referenced and used as:
[FindByColumnIndex(6), FindByCss("select[data-unique-identifier].status")]
[SelectsOptionByAttribute("value")]
public Select<BringUpStatusType, TOwner> StatusForBringUp { get; set; }
```

However, when I try to change the value of a given `<select>`, using the [Set()](https://github.com/atata-framework/atata/blob/925219c2e323594e50c49c4734dec82fe603d5fb/src/Atata/Components/Fields/EditableField%602.cs#L67) method I get the following error:

```
OpenQA.Selenium.NoSuchElementException: 'Unable to locate element:
- By: XPath ".//option[normalize-space(@value) = 'TODO/To do']"
- Search time: 7.022s
- Search options: {Visibility=Any, Timeout=7s, RetryInterval=0.25s, IsSafely=False}

Context element:
- Tag: select
- Location: {X=877, Y=696}
- Size: {Width=105, Height=22}
- Element ID: 29C0372F6690CAEC11D11B975AD5A6B2_element_96
- Text:
To do,
Completed,
Waived; 

For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception'
```

So I was able to fix it by updating the enum:

```C#
public enum BringUpStatusType
{
  [Term(TermMatch.Equals, "TODO")]
  [Term(TermMatch.Equals, "To do")]
  ToDo,
  [Term(TermMatch.Equals, "COMPLETED")]
  [Term(TermMatch.Equals, nameof(Completed))]
  Completed,
  [Term(TermMatch.Equals, "WAIVED")]
  [Term(TermMatch.Equals, nameof(Waived))]
  Waived
}
```

And it works with the above change, but I wanted to log this issue anyway as the error message reported an XPath value of `.//option[normalize-space(@value) = 'TODO/To do']`, which seems kind of wrong? 

`[Term]` accepts multiple `string` values which I just assumed meant it would search using something like LINQ's `.Any()`, but it's concatenating the values with a `/`.

**Comment by YevgeniyShunevych** [comment]
Hi @mamift,

Yes, you are right, that's a bug. I will provide a fix for multiple enum terms, like `[Term(TermMatch.Equals, "TODO", "To do")]`.

However the following usage should not work the way you expect:

```cs
[Term(TermMatch.Equals, "TODO")]
[Term(TermMatch.Equals, "To do")]
ToDo,
```

Atata always take only one first `TermAttribute`, so the second one will not be considered at all.

Also, if `option` values can differ, you can maybe rely on `[SelectsOptionByText]` if text values for all `select` elements are equivalent. In this case you will not need multiple terms.

**Comment by mamift** [comment]
Oh you're right, that didn't fix it - it just suppressed the error. The Title cased terms *"To do"* were for an older version of the webpage, and yes, the second `[Term]` didn't match. Anyway, I got around it by creating a new enum, and new model types.

**Comment by YevgeniyShunevych** [comment]
@mamift, the fix has been released in Atata v2.14.0.
